service/tablet_allocator.cc

```diff
@@ -699,6 +700,9 @@ class load_balancer {
         const locator::topology& topo = _tm->get_topology();
         migration_plan plan;
 
+        // Prepare table-wide resize plans first, since DC plan might have something to merge into the resize plan.
```

Comment:
What might DC plan merge into the resize plan? Why can't we merge resize plan into the DC plan? Merging should be commutative, and DC plan making doesn't have visibility into the resize plan.